src/book/02-system/04-other/01-nubook.mdx

```diff
@@ -5,3 +5,275 @@ title: NUbook
 description: The NUbots team handbook and documentation.
 slug: /system/nubook
 ---
+
+NUbook is the handbook and high-level documentation for the NUbots team. What you are looking at right now is NUbook! This page has information on how NUbook works.
+
+## Getting Started
+
+### Install Prerequisites
+
+You'll need [Git](https://git-scm.com/), [Node.js](https://nodejs.org/en/) and [Yarn](https://classic.yarnpkg.com/en/) installed.
+
+<details>
+<summary>Ubuntu installation instructions</summary>
+
+1. Install Git by running the following commands:
+
+   ```
+   sudo apt update
+   sudo apt install git
+   ```
+
+2. Install the latest version of Node.js by following the instructions [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) to (pick the latest version).
+3. Install the latest version of Yarn by following the instructions [here](https://classic.yarnpkg.com/en/docs/install#debian-stable).
+
+</details>
+
+<details>
+<summary>Windows installation instructions</summary>
+
+1. Download and install Git from <https://git-scm.com/download/win>
+2. Download and install Node.js from <https://nodejs.org/en/> (pick the LTS version)
+3. Download and install Yarn from <https://classic.yarnpkg.com/latest.msi>
+
+</details>
+
+<details>
+<summary>macOS installation installation</summary>
+
+1. Install Homebrew by following the instructions at <https://brew.sh/>
+2. Install Git, Node.js and Yarn
+
+   ```sh
+   brew upgrade
+   brew install git node && brew install yarn
+   ```
+
+</details>
+
+### Install NUbook
```

Comment:
How come we have separate instructions for installing prerequisites on different operating systems, and not installing NUbook. You can only use these instructions on windows if you emulate a linux shell can't you? Should we make that clear?

Reply:
The Windows instructions has a Git download, pretty sure that downloads Git Bash, which then can be used to run those commands. Might be worth specifying though.

Reply:
I think it is